packages/labs/eleventy-plugin-lit/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@lit-labs/eleventy-plugin-lit",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Eleventy plugin for rendering Lit components.",
+  "license": "BSD-3-Clause",
+  "homepage": "https://lit-element.polymer-project.org/",
+  "repository": "lit-html",
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "exports": {
+    ".": {
+      "default": "./index.js"
+    }
+  },
+  "files": [
+    "/src/",
+    "!/src/test/",
+    "/index.{d.ts,d.ts.map,js,js.map}"
+  ],
+  "scripts": {
+    "build": "npm run clean && npm run build:ts",
+    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
+    "build:ts:watch": "tsc --build --watch",
+    "build:demo": "cd demo && eleventy",
```

Comment:
Maybe add a section in the README on building and running the demo?

Reply:
Will do in a followup.